README.yaml

```diff
@@ -73,7 +72,7 @@ usage: |-
 
   ```hcl
   module "example" {
-    source = "https://github.com/cloudposse/terraform-example-module.git?ref=master"
+    source  = "https://github.com/cloudposse/terraform-helm-release.git?ref=master"
     example = "Hello world!"
```

Comment:
remove `example`, add the required attributes

Reply:
Completed.